content/en/docs/tasks/administer-cluster/nodelocaldns.md

```diff
@@ -47,18 +47,37 @@ This is the path followed by DNS Queries after NodeLocal DNSCache is enabled:
 {{< figure src="/images/docs/nodelocaldns.jpg" alt="NodeLocal DNSCache flow" title="Nodelocal DNSCache flow" caption="This image shows how NodeLocal DNSCache handles DNS queries." >}}
 
 ## Configuration
-
-This feature can be enabled using the command:
-
-`KUBE_ENABLE_NODELOCAL_DNS=true kubetest --up`
-
-This works for e2e clusters created on GCE. On all other environments, the following steps will setup NodeLocal DNSCache:
-
-* A yaml similar to [this](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml) can be applied using `kubectl create -f` command.
-* No need to modify the --cluster-dns flag since NodeLocal DNSCache listens on both the kube-dns service IP as well as a link-local IP (169.254.20.10 by default)
+NOTE: The local listen IP address for NodeLocal DNSCache can be any IP in the 169.254.20.0/16 space or any other IP address that can be guaranteed to not collide with any existing IP. We will use 169.254.20.10 as the default in the example below.
+
+This feature can be enabled using the following steps:
+* Prepare a yaml similar to [this](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml), save as nodelocaldns.yaml.
```

Comment:
Hello @prameshj .
I read through the changes, here is some non-technical feedback about the changes to the `Configuration` section:
Add a new line at line 53 so that the list renders correctly.
Should the reader manually edit the list of variables in their copy of `nodelocaldns.yaml`?
Then, run one of the `sed` scripts to substitute/set the other variables?
Line 59,  Could you add more information about the `sed` script and the variables in the script?

Reply:
Thanks for the review Karen.

Yes, reader sets variables and runs the "sed" command in bash. "sed" is a common command in bash, does it need additional documentation here?